cpp/src/io/parquet/page_enc.cu

```diff
@@ -1591,7 +1591,9 @@ __device__ void finish_page_encode(state_buf* s,
     }
     pages[blockIdx.x] = s->page;
     if (not comp_results.empty()) {
-      comp_results[blockIdx.x]   = {0, compression_status::FAILURE};
+      auto const status =
+        s->col.skip_compression ? compression_status::SKIPPED : compression_status::FAILURE;
```

Comment:
neat. the adapter hack to skip huge pages seems to be more useful that planned :)

Reply:
Yes, that was awesome :pray: